### DISCUSSION TITLE
Variables for passwords

### DISCUSSION THREAD

**Discussion by pejahlo** [Eugeny/tabby#9305] [General]
Hello, would it be possible to store different accounts (username+password) as variable so I do not have to change pwd upon each pwd update? Thank you. Peter

**Comment by ianaflous** [comment]
Hello, I suggested this #10076. It prevents saving the same credentials in many profiles. It has been merged, it's available in the [Latest release](https://github.com/Eugeny/tabby/releases/latest).
